kvariants/build.rs

```diff
@@ -0,0 +1,38 @@
+use serde::Deserialize;
+use std::error::Error;
+
+#[derive(Deserialize)]
+pub struct TsvRow {
+    lhs: String,
+    relation: String,
+    rhs: String,
+}
+
+fn main() {
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    println!("cargo:rerun-if-changed=dictionaries/source/kVariants.tsv");
+
+    compress_kvariant_txt().unwrap();
+}
+
+fn compress_kvariant_txt() -> Result<(), Box<dyn Error>> {
+    let reader = csv::ReaderBuilder::new()
+        .delimiter(b'\t')
+        .has_headers(false)
+        .from_path("dictionaries/source/kVariants.tsv");
+
+    let mut writer = csv::Writer::from_path("dictionaries/compressed/kVariants.csv")?;
```

Comment:
We may store the compressed dictionary in the target dir instead, which will avoid keeping an empty `compressed` directory in the repository.
```suggestion
    let out_dir = env::var_os("OUT_DIR").unwrap();
    let dst_path = Path::from(out_dir).join("kVariants.csv");
    let mut writer = csv::Writer::from_path(dst_path)?;
```

Reply:
Thanks! I wasn't aware of `OUT_DIR`. Fixed in 2c326cd.